tests/ui/apps/IncrementSlider/IncrementSlider-View.js

```diff
@@ -0,0 +1,21 @@
+import Heading from '../../../../Heading';
+import IncrementSlider from '../../../../IncrementSlider';
+import ThemeDecorator from '../../../../ThemeDecorator';
+import React from 'react';
+
+const app = (props) => <div {...props}>
+	<div>
+		<Heading>IncrementSlider default</Heading>
+		<IncrementSlider defaultValue={0} className="incrementSliderDefault" />
```

Comment:
It may be trivial but I think it is not meaningful to set defaultValue to 0. becase the default value of min prop is set to 0 in agate::IncrementSliders and set to value of value prop in ui::Slider. (refer to https://github.com/enactjs/enact/blob/master/packages/ui/Slider/Slider.js#L282) 
If you change the value, I think  `defaultValue` prop is meaningful.